src/commands/coffeechat/coffeeTest.ts

```diff
@@ -0,0 +1,27 @@
+import { Message, MessageEmbed } from 'discord.js';
```

Comment:
Test func is a great idea :)
Can you add a printout of all the matches? ie 5 ppl would give smth like
3 4
1 5
2

Reply:
I think the purpose of the `coffeetest` command was just to determine the number of rounds before there is a duplicate. As such, I guess it would be quite a lengthy printout, since it would be a sequence of what you have described above. Would a printout be necessary?

Reply:
Oh, I didn't understand the purpose. But are we able to see entire match-sets anywhere else so far?
Maybe @alexz429 can talk abt this more since he coded this stuff.

Reply:
As for seeing the results, currently, you cannot see them for `coffeetest`, while you can view historic matches in the `coffee_historic_matches` table for `coffeematch`. Note that for `coffeematch`, the matches are not precomputed.

Reply:
viewing match sets is not supported anywhere right now: the overall reasoning was just that, at a certain point, to view match sets with > 10 people would cause half a channel to be filled with a massive embed. 

Technically, viewing db contents can let you piece together what the matches were I guess.